main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -773,6 +773,7 @@ Resources:
               - 'arn:aws:iam::*:role/*-xacc-env-mgmt'
               - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${self:custom.settings.namespace}-prep-raas-master-MasterRole-*'
               - 'arn:aws:iam::*:role/${self:custom.settings.globalNamespace}-DevopsRole'
+              - 'arn:aws:iam::*:role/*-cfn-status-role'
```

Comment:
Why does this role need to be assumed now?

Reply:
The workflow loop runner lambda gets triggered when a workspace is being provisioned. Until now workspace provisioning did not need to search for VPC endpoint IDs from hosting accounts, but after this change it will. The `cfn-status-role` in hosting account allows the app to get this information.